core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.scss

```diff
@@ -26,31 +28,53 @@
             font-size: $font-size-sm;
             margin: $spacing-3;
         }
-    }
 
-    .p-dropdown {
-        display: flex;
-        flex-direction: row-reverse; // this allows having the plus icon at the left of the placeholder
-        border: none;
-        background: none;
-        font-size: $spacing-2;
-        margin-left: $spacing-4;
-        padding: $spacing-2;
-    }
-    .p-dropdown:hover {
-        background: $color-palette-primary-200;
-    }
-    .p-dropdown-label.p-inputtext {
-        font-size: $font-size-sm;
-        color: $black;
-        padding: 0 0 0 $spacing-2;
-        max-width: 10rem;
+        .p-dropdown .p-dropdown-trigger {
+            width: 0.875rem; // 14px
+            height: 0.875rem; // 14px
```

Comment:
This should be global.

Reply:
This is to fix the width and height when the component is small, so it looks like a regular button